internal/generate/servicepackage/main.go

```diff
@@ -512,6 +519,34 @@ func (v *visitor) processFuncDecl(funcDecl *ast.FuncDecl) {
 			}
 
 			switch annotationName := m[1]; annotationName {
+			case "Action":
+				if len(args.Positional) == 0 {
+					v.errs = append(v.errs, fmt.Errorf("no type name: %s", fmt.Sprintf("%s.%s", v.packageName, v.functionName)))
+					continue
+				}
+
+				typeName := args.Positional[0]
+
+				if !validTypeName.MatchString(typeName) {
+					v.errs = append(v.errs, fmt.Errorf("invalid type name (%s): %s", typeName, fmt.Sprintf("%s.%s", v.packageName, v.functionName)))
+					continue
+				}
+
+				if d.Name == "" {
+					v.errs = append(v.errs, fmt.Errorf("no friendly name: %s", fmt.Sprintf("%s.%s", v.packageName, v.functionName)))
+					continue
+				}
+
+				if _, ok := v.actions[typeName]; ok {
+					v.errs = append(v.errs, fmt.Errorf("duplicate Action (%s): %s", typeName, fmt.Sprintf("%s.%s", v.packageName, v.functionName)))
+				} else {
+					v.actions[typeName] = d
+				}
+
+				if d.HasV6_0SDKv2Fix {
+					v.errs = append(v.errs, fmt.Errorf("V60SDKv2Fix not supported for Actions: %s", fmt.Sprintf("%s.%s", v.packageName, v.functionName)))
+				}
```

Comment:
Is there a reason we check and error only on this annotation? It seems there could be many which don't apply to actions (`SingletonIdentity`, `WrappedImport`, etc.).

Reply:
You're absolutely right. The `V60SDKv2Fix` annotation was specifically created to address a v6.0 provider bug related to the Resource Identity feature that only affected certain SDK-based resources. Since Actions are a new resource type that wasn't affected by that bug, this check is unnecessary.

Looking at the broader pattern, if we want to validate unsupported annotations for Actions, we should either:
- Validate ALL annotations that don't apply to Actions (not just V60SDKv2Fix), or
- Remove this specific check since it's not needed

I'll remove the V60SDKv2Fix check for Actions since it's not applicable.